dev/cnf/oss_ibm.maven

```diff
@@ -1,6 +1,6 @@
 com.googlecode.jsontoken:jsontoken:1.1.0-ibm-s20140416-2113
 com.ibm.ws.com.icegreen:greenmail:2.0.0.fd826131f671b0d3230b0d0e1638d8c4
-com.ibm.ws.commons-collections:commons-collections:3.2.1
+com.ibm.ws.commons-collections:commons-collections:3.2.2
```

Comment:
@cbridgha  Does this need to be updated? 

It looks like the 3.2.1 is linked to https://public.dhe.ibm.com/ibmdl/export/pub/software/olrepo/com/ibm/ws/commons-collections/commons-collections/3.2.1/ 

How could I upload my 3.2.2 jar, if needed?

Reply:
I can relocate the jars in Artifactory and DHE if you need me to.